README.md

```diff
@@ -39,7 +40,7 @@ import (
 func main() {
 	defer signal.Subscribe(func(signal os.Signal) {
 		fmt.Println("this code is execute when signal os.Interrupt, os.Kill was received")
-	}, os.Interrupt, os.Kill)()
+	}, os.Interrupt, syscall.SIGTERM)()
```

Comment:
the printed message **l42** is incorrect then no ?

Reply:
yes